core/src/main/java/org/web3j/ens/EnsMetadataResponse.java

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright 2024 Web3 Labs Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package org.web3j.ens;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class EnsMetadataResponse {
+    public boolean is_normalized;
+    public String name;
+    public String description;
+    public Attribute[] attributes;
+    public String url;
+    public long last_request_date;
+    public int version;
+    public String background_image;
+    public String image;
+    public String image_url;
+
+    public boolean isIs_normalized() {
```

Comment:
I think we should start to use Lombok in oder to avoid all these setters and getters

Reply:
I agree, I can include Lombok in another PR and update all the classes